### DISCUSSION TITLE
Counter-current fire spreads(PMMA)

### DISCUSSION THREAD

**Discussion by liuyue388** [firemodels/fds#13830] [Q&A]
There are two bugs with this procedure，how should I fix these problems

[model_git.txt](https://github.com/user-attachments/files/18001718/model_git.txt)

**Comment by drjfloyd** [comment]
What bugs?
What procedure?

You haven't provided any information that can be used to assist you.

**Reply by liuyue388** [reply]
Thank you for your reply.
Here's my procedure.
1. &MESH XB = 0 1 0 6 0 1 
2. IJK =  100 600 100 /
3. &TIME T_END = 500 /
4. &MISC TMPA =  20 
5. &VENT MB = YMAX SURF_ID = 'OPEN'/
6. &SURF ID = 'wind' 
7.       VEL = -5    
8.       COLOR = 'BLUE' /
9. &VENT XB = 0 1 0 0 0 1
10.       SURF_ID = 'wind' /   
11. &SPEC ID = 'MMA'
12.       FORMULA = 'C5.0H8.0O2.0' /
13. &SPEC ID = 'OXYGEN'          LUMPED_COMPONENT_ONLY = T /
14. &SPEC ID = 'NITROGEN'        LUMPED_COMPONENT_ONLY = T / 
15. &SPEC ID = 'WATER VAPOR'     LUMPED_COMPONENT_ONLY = T /
16. &SPEC ID = 'CARBON DIOXIDE'  LUMPED_COMPONENT_ONLY = T / 
17. &SPEC ID = 'AIR'  BACKGROUND = T  
18.       SPEC_ID(1) = 'OXYGEN' VOLUME_FRACTION(1) = 6
19.       SPEC_ID(2) = 'NITROGEN' VOLUME_FRACTION(2) = 19.8621  /
20. &SPEC ID = 'PRODUCTS' 
21.       SPEC_ID (1) = 'CARBON DIOXIDE' 
22.       VOLUME_FRACTION (1) = 5
23.       SPEC_ID (2) = 'WATER VAPOR'     
24.       VOLUME_FRACTION (2) = 4
25.       SPEC_ID (3) = 'NITROGEN'        
26.       VOLUME_FRACTION (3) = 19.8621 /
27. &REAC FUEL = 'MMA'
28.       HEAT_OF_COMBUSTION = 25200
29.       SPEC_ID_NU = 'MMA','AIR','PRODUCTS' 
30.       NU = -1,-1,1 /       
31. &MATL ID = 'PMMA'
32.       SPECIFIC_HEAT_RAMP = 'PMMA_SPECHEAT_FUNCTION'
33.       CONDUCTIVITY_RAMP  = 'PMMA_CONDUCT_FUNCTION'
34.       REFRACTIVE_INDEX = 1.576
35.       DENSITY = 1160
36.       ABSORPTION_COEFFICIENT = 2250
37.       EMISSIVITY = 0.95
38.       A(1) = 8.60E12
39.       E(1) = 188000
40.       SPEC_ID = 'MMA'
41.       N_REACTIONS = 1
42.       NU_SPEC = 1.0
43.       HEAT_OF_REACTION = 846 /
44. &RAMP ID = 'PMMA_SPECHEAT_FUNCTION'  T = 27     F = 1.70 /
45. &RAMP ID = 'PMMA_SPECHEAT_FUNCTION'  T = 107    F = 2.03 /
46. &RAMP ID = 'PMMA_SPECHEAT_FUNCTION'  T = 187    F = 2.32 /
47. &RAMP ID = 'PMMA_SPECHEAT_FUNCTION'  T = 267    F = 2.61 /
48. &RAMP ID = 'PMMA_SPECHEAT_FUNCTION'  T = 337    F = 2.87 /
49. &RAMP ID = 'PMMA_SPECHEAT_FUNCTION'  T = 407    F = 3.13 /
50. &RAMP ID = 'PMMA_CONDUCT_FUNCTION'   T = 27     F = 0.33 /
51. &RAMP ID = 'PMMA_CONDUCT_FUNCTION'   T = 107    F = 0.20 /
52. &RAMP ID = 'PMMA_CONDUCT_FUNCTION'   T = 187    F = 0.15 /
53. &RAMP ID = 'PMMA_CONDUCT_FUNCTION'   T = 267    F = 0.14 /
54. &RAMP ID = 'PMMA_CONDUCT_FUNCTION'   T = 337    F = 0.12 /
55. &RAMP ID = 'PMMA_CONDUCT_FUNCTION'   T = 407    F = 0.10 /
56. &SURF ID = 'PMMA CUBE'
57.       BURN_AWAY = .TRUE.
58.       MATL_ID = 'PMMA'
59.       BACKING = 'INSULATED'
60.       THICKNESS = 0.1 /     
61. &OBST XB = 0.4 0.6 3 4 0 0.1
62.       SURF_ID = 'PMMA CUBE'
63.       RGB = 128 0 64/     
64. &OBST XB = = 0 0.4 0 6 0 0.1
65.       SURF_ID = 'INERT'
66.       RGB = 192 192 192 /
67. &OBST XB = 0.6 1 0 6 0 0.1
68.       SURF_ID = 'INERT'
69.       RGB = 192 192 192 /
70. &OBST XB = 0.4 0.6 0 3 0 0.1
71.       SURF_ID = 'INERT'
72.       RGB = 192 192 192 / 
73. &OBST XB = 0.4 0.6 4 6 0 0.1
74.       SURF_ID = 'INERT'
75.       RGB = 192 192 192 / 
76. &SLCF PBY = 3.5
77.       QUANTITY = 'TEMPERATURE'
78.       VECTOR = T /
79. &SLCF PBY = 3.5
80.       QUANTITY = 'VELOCITY'
81.       VECTOR = T /
82. &SLCF PBY = 3.5
83.       QUANTITY = 'MASS FRACTION'
84.       SPEC_ID = 'CARBON DIOXIDE'
85.       VECTOR = .TRUE. /                                            
86. &SLCF PBY = 3.5
87.       QUANTITY = 'MASS FRACTION'
88.       SPEC_ID = 'MMA'
89.       VECTOR = .TRUE. / 
90. &SLCF PBY = 3.5
91.       QUANTITY = 'MASS FRACTION'
92.       SPEC_ID = 'OXYGEN'
93.       VECTOR = .TRUE. / 
94. &DEVC ID = 'TIMER'     XYZ = 0.55 2.5 0.15     SETPOINT = 30   
95.                        QUANTITY = 'TIME'   INITIAL_STATE = T /
96. &SURF ID = 'IGNITER'
97.       TMP_FRONT = 2500
98.       RAMP_T = 'tmp_ramp'
99.       COLOR = 'RED' /
100. &RAMP ID = 'tmp_ramp'   T = 0     F = 1.000 /
101. &RAMP ID = 'tmp_ramp'   T = 15    F = 1.000 /
102. &RAMP ID = 'tmp_ramp'   T = 20    F = 0.640 /    
103. &RAMP ID = 'tmp_ramp'   T = 25    F = 0.360 /
104. &RAMP ID = 'tmp_ramp'   T = 30    F = 0.160 /
105. &OBST XB = 0.55 0.6 0 6 0.1 0.15
106.       DEVC_ID = 'TIMER'
107.       SURF_ID6 = 'IGNITER','INERT','INERT','INERT','IGNITER','INERT' /
108. &TAIL /

There are two errors with this procedure.
One error is ：Problem with OBST number 2, line number 62 
Another is : DEVICE Activation Times 
                    1 TIMER         No  Activation
I don't know how to fix these errors, thank you so much for helping me.

**Comment by drjfloyd** [comment]
4.2 Common Error Statements

An FDS calculation may end before the specified time limit. Following is a list of common error statements
and how to diagnose the problems:

Input File Errors: The most common errors in FDS are due to mis-typed input statements. These errors
result in the immediate halting of the program and a statement like, “ERROR: Problem with the HEAD
line.” For these errors, check the line in the input file named in the error statement. Make sure the
parameter names are spelled correctly. Make sure that a / (forward slash) is put at the end of each
namelist entry. Make sure that the correct type of input data is provided for each input parameter, such
as one real number, several integers, or something else. Make sure there are no non-ASCII characters
being used, as can sometimes happen when text is cut and pasted from other applications or wordprocessing
software. Make sure that the number zero (0) is not typed as the capital letter O and vice
versa. Make sure that the number one (1) is not an exclamation point (!) or lower case letter l. Make sure
apostrophes are used to designate character strings. Make sure the text file on a Unix/Linux machine
was not created on aWindows machine, and vice versa. Make sure that all the parameters listed are still
being used – new versions of FDS often drop or change parameters forcing you to re-examine old input
files.

**Reply by liuyue388** [reply]
Thank you very much for your help, I will try to fix these errors.
